cornichon-http-mock/src/main/scala/com/github/agourlay/cornichon/http/server/MockServerRequestHandler.scala

```diff
@@ -36,36 +37,54 @@ class MockServerRequestHandler() {
       Ok(body)
 
     case GET -> Root / "reset" ⇒
-      requestReceivedRepo.clearRegisteredRequest()
+      mockState.clearRegisteredRequest()
       Ok()
 
     case r @ POST -> Root / "response" ⇒
       r.bodyAsText.runFold("")(_ ++ _).flatMap { body ⇒
-        requestReceivedRepo.setResponse(body)
+        mockState.setResponse(body)
         Ok()
       }
 
+    case r @ POST -> Root / "delayInMs" ⇒
```

Comment:
So the "API" would be:
```
When I post("<mock-url>/delayInMs").withBody("250")
```
❓

Reply:
exactly, I tested manually but could not write a proper cornichon test for it as there is no assertion on the response time...yet 😄